src/components/recurring-donation/Form.tsx

```diff
@@ -102,9 +95,30 @@ export default function EditForm() {
     },
   })
   async function onSubmit(data: RecurringDonationInput) {
+    data.amount = toMoney(data.money)
+    data.sourceVault = data.vaultId
+    delete data.money
+    delete data.campaign
```

Comment:
Can't we omit the data.money and data.campaign and populate them with the needed values based on the form field names?

Reply:
Well, in the edit form money and amount is different: 12.00 vs 1200. 
It seems cleaner this way, since we don't mix the two.